foundation-extensions/mobile-core/rules-engine/README.md

```diff
@@ -49,11 +49,13 @@ At the start of a new application session that includes the Experience Platform
 
 On older versions of Experience Platform Mobile SDKs \(prior to iOS version 1.6.2, Android version 1.5.4\), after the first launch of the app, it always takes some time to download the rules from the remote servers. During this time, Rules Engine won't be able to evaluate the first several events until the rules are loaded. Starting from iOS version 1.6.2 and Android version 1.5.4, we add the capability to cache the events before the rules are downloaded and will re-evaluate those events afterward. This change is mainly to enable the trigger of Posback based on the install event.
 
-### Using Bundled Rules (iOS AEP 3.x Only)
+### Using Bundled Rules
 
-In addition to the remote configuration, you can also include a rules zip file in your app bundle to be used by the SDK before rules have been downloaded from the Data Collection UI. To add bundled rules to your app, follow these steps:
+In addition to the remote configuration, you can also include a rules zip file in your app to be used by the SDK before rules have been downloaded from the Data Collection UI. This feature is supported in iOS SDK version 3.x, Android SDK version 1.11.2 onwards. To add bundled rules to your app, follow these steps:
```

Comment:
I think instead of adding the version requirement inside the docs, we should add it to the header. What do you think @praveek @emdobrin?

Reply:
Hmm, putting it solely in the header would be a bit noisy, IMO.

Maybe have it as an important tag under the header instead?

Reply:
I added the support version details as a info hint similar to one in [configuration docs](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration#environment-aware-configuration-properties).